spec/controllers/admin/server_recordings_controller_spec.rb

```diff
@@ -57,4 +72,22 @@
       end
     end
   end
+
+  describe '#resync' do
+    it 'calls the RecordingsSync service correctly' do
+      expect_any_instance_of(RecordingsSync).to receive(:call)
+      get :resync
+    end
+
+    it 'calls the RecordingsSync service with correct params' do
+      expect(RecordingsSync).to receive(:new).with(user:)
```

Comment:
After asserting the response this will fail, we should either call the original `:new` method implantation or stub the the `:new` method returning an instance double (verified double) when we don't want to trigger the original implentation,

Reply:
I think all this stuff should be done in a different PR. I only moved the resync method since it affected the ManageRecordings permission

Could you explain this one more though, I don't understand why it's failing